### PR TITLE
chore(e2e): disable range test for 8.0+

### PR DESCRIPTION
**Pull request by paula-stacho**
### Checklist
- [ ] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [ ] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/compass-e2e-tests/tests/in-use-encryption.test.ts [hunk 1]
```diff
@@ -447,6 +447,10 @@ describe('CSFLE / QE', function () {
         ['range', collectionNameRange],
       ] as const) {
         it(`can edit and query the ${mode} encrypted field in the CRUD view`, async function () {
+          // TODO: re-enable after 7.3/8.0 support is ready
```
Comment (addaleax) [inline packages/compass-e2e-tests/tests/in-use-encryption.test.ts:450]:
```suggestion
          // TODO(COMPASS-7760): re-enable after 7.3/8.0 support is ready
```

**Review by addaleax — APPROVED** [review]

**Review by addaleax — APPROVED** [review]
@lerouxb the ticket number is for re-enabling the test, not for disabling it, no?

**Comment by paula-stacho** [comment]
looks like the test is not skipped 🙈 https://spruce.mongodb.com/task/10gen_compass_main_ubuntu_test_packaged_app_latest_patch_1a8fae4cd761fb95c2e66cd283df5868d08b68a1_65f428fc9d401900072b8c32_24_03_15_10_54_54?execution=0&sortBy=STATUS&sortDir=ASC
I don't see anything wrong with the condition, I can only try to add some logs

**Comment by paula-stacho** [comment]
Changing the condition to `>7`, looks like the current version is `8.0.0-alpha-5224-g23a89a3` which is considered lower than 8.

**Review by himanshusinghs — APPROVED** [review]

**Pull request merged**